### DISCUSSION TITLE
the v0.42.2 has not upload foldes

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#182] [General]
the version 0.42.2 works great. but sorry, I just want to expand the story. the version 0.42.2 now does not contain an upload folders. I need in addition to uploading filtered files per extension and uploading folders with filtered files per extension
![upload folders](https://user-images.githubusercontent.com/21057856/228331388-30cb5971-e4f0-4e94-a272-592b7d64590f.jpg)

**Comment by rejetto** [comment]
it HAS upload folders.
it's just not supported at the same time with the "accept" filter.
[The browser does NOT support folder + filter](https://bugs.chromium.org/p/chromium/issues/detail?id=102984).

I have a partial solution:
I can filter the list AFTER the selection, but this means the browser, when the folder has 2 correct files on a total of 6 files, will show a message reporting the total amount of files
<img width="208" alt="image" src="https://user-images.githubusercontent.com/1367199/228564716-db46a64b-8c98-4ead-a1df-fb9cab264f10.png">
and THEN i will put 2 files in the list.

Ok? What you think?

**Comment by rejetto** [comment]
please notice that when you send me these screenshots, i cannot read serbian.
Consider making screenshots with ?lang=en in the address for a temporary change
